src/components/ExpansionCardFormField.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode, useEffect, useState } from "react";
+import { ExpansionCard } from "@navikt/ds-react";
+
+interface Props {
+  ariaLabel: string;
+  heading: ReactNode;
+  children: ReactNode;
+  hasError?: boolean;
+}
+
+export const ExpansionCardFormField = ({
```

Comment:
Skal vi kjøre `function` som standard? 😊
```suggestion
export function ExpansionCardFormField({
```

Reply:
Ja, det er nok en del steder som må endres da, kan ta det i en videre PR for å rydde opp i styling.